mobile/events/events.tsx

```diff
@@ -15,10 +15,11 @@ export type Event = {
   url?: string
 }
 
+const SQUAD_CODE_TITLE_TEXT = "Squad Code: "
 
 const Events = (props) => {
   const [events, setEvents] = useState([])
-  const [squadId, setSquadId] = useState(props.route.params.squadId)
```

Comment:
We still need the squadId to retrieve events so I don't think you should remove this. Squad code should be different.

Reply:
removed